rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java

```diff
@@ -31,4 +31,12 @@
 
     Single<Map<String, Long>> mapCountriesToNamePopulation(List<Country> countries);
 
+    Observable<Long> combineAndReturnTheSumOfPopulation(Observable<Country> countryObservable1,
+                                                        Observable<Country> countryObservable2);
+
+    Single<Boolean> checkIfBothObservableEmitsSameItems(Observable<Country> countryObservable1,
+                                                        Observable<Country> countryObservable2);
+
+    Single<Boolean> checkIfObservableEmitsGivenItem(List<Country> countries,
```

Comment:
Actually no, i have added it just to test the 'contains' operator functionality.